target/NeighborHelp-1/admin_dashboard.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
```

Comment:
This is probably alright for now, but you might want to add a .gitignore file to ignore the /target folder when doing commits.  Checkout the one that's currently on the github master branch, if you want you can just copy that into your /Neighborgood folder.

Reply:
Sounds like a swell idea!